### ISSUE TITLE
Config for custom datasets

### ISSUE THREAD

**Issue by AaspiralMoon** [hasanirtiza/Pedestron#173]
Hello, I'm wondering how to prepare the training config for custom datasets. Is there any detailed instructions to follow? Since there are many parameters to tune, and I don't quite get what they represent.

Thanks!

**Comment by hasanirtiza** [comment]
I would start by looking into configs/elephant folder, simply convert your dataset to coco format first, and run the config to see how far it is. The usual parameters learning rate etc (starting ones) are in the config file under optimizers (optimizer = dict(type='SGD', lr=0.02, momentum=0.9, weight_decay=0.0001)) etc. Secondly, since this work is based on earlier version of mmdetection but a few of the params and their meaning are essentially same, you can have a brief look at what config file means on the new documentaion, although some of the things have been changed
(https://mmdetection.readthedocs.io/en/dev-3.x/user_guides/config.html) or have a look at the technical report (https://arxiv.org/pdf/1906.07155).
